src/hdrgen.c

```diff
@@ -1692,10 +1696,13 @@ class PrettyPrintVisitor : public Visitor
         }
         else
         {
-            buf->writestring(d->ident->toChars());
+            const char* id = d->ident->toChars();
+            declstring = (strcmp (id, "string") == 0 || strcmp (id, "wstring") == 0 || strcmp (id, "dstring") == 0);
```

Comment:
Added the string types to idgen. I'm quite surprised that these are not in there yet.